src/vscode-dts/vscode.d.ts

```diff
@@ -19017,9 +19020,6 @@ declare module 'vscode' {
 		 *   console.error(e);
 		 * }
 		 * ```
-		 *
```

Comment:
Likely happening because everything after the `@example` until the next `@` tag is considered part of the example

A little unclear if that's expected or not for jsdoc/tsdoc. The two specs sometimes don't agree